docs/cluster-setup-guide/deploy-cluster/sysadmin-deploy-on-slurm/slurm-known-issues.rst

```diff
@@ -91,6 +92,61 @@ Some constraints are due to differences in behavior between Docker and Singulari
    | ``environment.registry_auth.email``  | No equivalent setting in Singularity.                |
    +--------------------------------------+------------------------------------------------------+
 
+*********************
+ PodMan Known Issues
+*********************
+
+   -  On a Slurm cluster, it is common to rely upon ``/etc/hosts`` to resolve the addresses of the
+      login node and other compute nodes in the cluster. Unlike Singularity, PodMan V4.0+ no longer
+      maps ``/etc/hosts`` from the host into the running container by default. If jobs are unable to
+      connect to the Determined master, or other compute nodes in the job, check the following:
+
+      #. If you are relying on ``/etc/hosts`` (instead of DNS) for address resolution, ensure that
+         the names and addresses of the login node, admin node, and all compute nodes are
+         consistently available in ``/etc/hosts`` on all nodes.
+
+      #. Provide an automatic mapping of ``/etc/hosts`` into the container by adding a :ref:`bind
+         mount <exp-bind-mounts>` to the ``task_container_defaults`` section of the
+         ``/etc/determined/master.yaml`` file as shown below. On the initial startup, the Determined
+         Slurm launcher automatically adds this section when adding the ``resource_manager``
```

Comment:
not clear to me as a reader what exactly this is trying to say

Reply:
Suggestions welcome as to how to clarify.

Reply:
It's clear to me, except that the opening paragraph says that Podman 4.0+ does not map "/etc/hosts" from the host onto the running container, and then the first bullet item says to ensure that nodes are available in "/etc/hosts".  One point of confusion might be why you are asking the user to check that the nodes are available in "/etc/hosts" when you just finished telling them that podman is not going map "/etc/hosts".  Secondly, the first bullet item should say that you are checking for correctness on the "/etc/hosts" of the host, so there's no confusion as to whether the paragraph is referring to the container or the host.

Perhaps the second bullet item should come first, but since the opening paragraph says "check the following:", the second paragraph should probably start with "If you are relying on '/etc/hosts' for address resolution, ensure that you've configured your 'master.yaml' to explicitly map '/etc/hosts' from the host onto the container.  This is accomplished by adding a ....".

Reply:
I guess i was just thinking this part specifically.

> adds this section when adding the ``resource_manager`` section

is this the ``resource_manager`` section of the launcher config that gets dynamically set from the determined stuff?

Reply:
Changed it to be more explicit:  

```
On the initial startup, the Determined Slurm launcher automatically adds the task_container_defaults fragment below when adding the resource_manager section. 
```